extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -424,6 +430,51 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Returns a promise that resolves when configuration is re-written if
+   * configRewriter is configured by a vendor.
+   *
+   * @private
+   * @return {!Promise<undefined>}
+   */
+  rewriteConfig_() {
+    const remoteConfigUrl = this.getConfigRewriter_()['url'];
+
+    if (!remoteConfigUrl || this.isSandbox_) {
+      // use default configuration merge.
+      return Promise.resolve();
+    }
+
+    assertHttpsUrl(remoteConfigUrl, this.element);
+    const TAG = this.getName_();
+    dev().fine(TAG, 'Rewriting config', remoteConfigUrl);
+
+    const config = dict({});
+    this.mergeObjects_(this.getInlineConfigNoInline(), config);
+    this.mergeObjects_(this.remoteConfig_, config);
+
+    const fetchConfig = {
+      requireAmpResponseSourceOrigin: false,
+      method: 'POST',
+      body: config,
+      credentials: 'include',
```

Comment:
do you really need user cookie here?

Reply:
Some services might have additional configuration stored in cookies (i.e. gtm debug/preview)
Should I make this configurable? (the way "data-credentials" attribute controls this for remote config).